marklogic-data-hub/src/test/java/com/marklogic/hub/HubTestBase.java

```diff
@@ -183,13 +183,15 @@ protected HubConfigImpl runAsUser(String mlUsername, String mlPassword) {
         applyMlUsernameAndMlPassword(mlUsername, mlPassword);
 
         // Re-initializes the Manage API connection
-        getHubConfig().getManageClient().setManageConfig(hubConfig.getManageConfig());
+        getHubConfig().getManageClient().setManageConfig(adminHubConfig.getManageConfig());
```

Comment:
We want the code on the left, not on the right.

Reply:
@rjrudin fixed and pushed, please review.